action.yml

```diff
@@ -1,8 +1,6 @@
 name: 'Create/destroy runners'
 description: 'Github action to create and destroy Runners'
-author:
-  - 'cellinacywinska'
-  - 'LorenzoDrudi'
+author: 'cellinacywinska && LorenzoDrudi'
```

Comment:
Ops, I leave the comment also here. I decided to do it in this way because the yaml vscode extension gave a warning with the previous way. I searched online but I didn't find any example/docs so I do that in the more appropriate way in my opinion.